sr_data_visualization/src/sr_data_visualization/data_plot.py

```diff
@@ -91,7 +89,7 @@ def timerEvent(self):
 
         self.replot()
 
-    def plot_data(self, plot, side, new_sub=True):
+    def plot_data(self, plot, side, new_sub=True):  # pylint: disable=W0613
```

Comment:
if new_sub is not used, why not remove this?

Reply:
Its still here to keep it consistent with the other plot_data, or it would throw errors here https://github.com/shadow-robot/sr-visualization/blob/noetic-devel/sr_data_visualization/src/sr_data_visualization/sr_data_visualizer_gui.py#L197